mongoimport.py

```diff
@@ -0,0 +1,36 @@
+import mongoengine
+from mongoengine import Document, fields
+from dictionaryScript import crawl
+import logging
+import os
+
+logger = logging.getLogger(__name__)
+
+class Word(Document):
+    word = fields.StringField(primary=True)
+    definition = fields.ListField(default=[])
+
+def mongo_import(host, db, username, password, alph):
```

Comment:
Thanks for going through the effort to pipe this into a formal database. I started this because I was trying to train a logistic regression to search for words by informal definitions. I ended up making it public because of how difficult it is to find a free and easy-to-access DB with a dictionary in it. I appreciate the value added here and it's clear you accomplished what you set out to do, so I will merge this. Definitely feel free to submit PRs to this in the future.

Reply:
Good efforts mate. 

I used open English WordNet for my word-squad game, it works but it's locked in MySQL. I'll switch it to MongoDB shortly.

Reply:
nice.